hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/testutils/HoodieClientTestBase.java

```diff
@@ -770,6 +770,7 @@ protected void updateBatchWithoutCommit(String newCommitTime, List<HoodieRecord>
     HoodieWriteConfig hoodieWriteConfig = getConfigBuilder(HoodieFailedWritesCleaningPolicy.LAZY)
         .withAutoCommit(false) // disable auto commit
         .withRollbackUsingMarkers(true)
+        .withHeartbeatTolerableMisses(0)
         .build();
```

Comment:
What the purpose of this change?

Reply:
@danny0405, accelerate the expiration of the heartbeat for the deltacommit and not wait for the heartbeat expiration in [line 308](https://github.com/apache/hudi/blob/master/hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/functional/TestSavepointRestoreMergeOnRead.java#L308) for `TestSavepointRestoreMergeOnRead#testCleaningCompletedRollback`.